service-base/src/main/java/org/eclipse/hono/service/registration/RegistrationAssertionHelperImpl.java

```diff
@@ -43,11 +52,11 @@ private RegistrationAssertionHelperImpl(final Vertx vertx) {
 
     /**
      * Creates a helper for creating registration assertions.
-     * 
-     * @param vertx The vertx instance to use for accessing the file system.
+     *
+     * @param vertx  The vertx instance to use for accessing the file system.
      * @param config The configuration properties to determine the signing key material from.
      * @return The helper.
-     * @throws NullPointerException if any of the params is {@code null}.
+     * @throws NullPointerException     if any of the params is {@code null}.
```

Comment:
please undo these formatting changes and adapt your code formatting rules to not arbitrarily change existing code you are nor making changes to